src/Contexts/cartContext.js

```diff
@@ -0,0 +1,17 @@
+import React, { createContext, useContext, useState } from 'react';
+
+const CartContext = createContext();
+
+const CartProvider = ({ children }) => {
+  const [items, setItems] = useState([]);
+
+  return (
+    <CartContext.Provider value={{ items, setItems }}>
```

Comment:
Instead of passing the setState function directly can  abstract it and pass it on.

```const setCartItem = () => setItem(items) ``` something like this

Reply:
ok